db/ddl.sql

```diff
@@ -78,5 +78,6 @@ CREATE TABLE `litle_responses` (
   `updated_at` datetime NOT NULL,
   `kb_account_id` varchar(255) DEFAULT NULL,
   `kb_tenant_id` varchar(255) DEFAULT NULL,
+  `approved_amount` int(11) DEFAULT NULL,
```

Comment:
What is this for / what does it represent?

If we need it, can you:

* Prefix it with `params_` (convention)
* Place it next to the other `params_` columns (somewhere near lines 63-69 above)
* Provide the [migration script](http://docs.killbill.io/latest/database_migrations.html#_ruby_plugins)

Also, is an `approved_currency` returned as well? If so, let's add it (could matter for cross-border transactions).